src/pattern-library/components/Library.tsx

```diff
@@ -63,20 +63,26 @@ function Heading({
 }: LibraryHeadingProps) {
   if (level <= 2) {
     return (
-      <h2 className="text-3xl text-slate-600 font-bold" {...htmlAttributes}>
+      <h2
+        className="text-3xl text-slate-600 font-bold mb-4"
```

Comment:
Margin fixes. If the smattering of changes in this module feel noisy, you might do better reviewing commit by commit.